tests/Doctrine/Tests/ORM/UnitOfWorkTest.php

```diff
@@ -541,7 +541,7 @@ public function testObjectHashesOfMergedEntitiesAreNotUsedInOriginalEntityDataMa
 
         $this->_unitOfWork->persist($newUser);
 
-        self::assertSame([], $this->_unitOfWork->getOriginalEntityData($newUser), 'No original data was stored');
+        self::assertEmpty($this->_unitOfWork->getOriginalEntityData($newUser), 'No original data was stored');
```

Comment:
We cannot do this yet since `UnitOfWork#getOriginalEntityData()` has no return type declaration, so we must ensure that we have the correct type.